### PR TITLE
Components package

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3256/commits/f5ac740eb59722269a5cb207c55de83bd495e44e failed.

File: packages/components/src/stories/decorators/ThemeDecorator.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (siddharthkp) [inline packages/components/src/stories/decorators/ThemeDecorator.tsx]: This is amazing! I didn't know it was possible to wrap each story in a themewrapper on it's own ✨

File: packages/components/src/design-language/typography.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export const fontSizes = [
```
Comment (siddharthkp) [inline packages/components/src/design-language/typography.ts]: Should we import this from `common/design-language` instead of copying it here?

File: packages/components/package.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "@codesandbox/common": "^1.0.8",
+    "@codesandbox/template-icons": "^1.1.0",
+    "@styled-system/css": "^5.1.4",
+    "codesandbox-api": "0.0.23",
+    "color": "3.1.2",
+    "date-fns": "^2.8.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-router-dom": "^5.1.2",
+    "styled-components": "^4.4.1",
```
Comment (siddharthkp) [inline packages/components/package.json]: oh my god, are you saying we can use a different styled-component version/library than the app? Is that a thing?

File: packages/components/.storybook/config.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+import { withKnobs } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import { addDecorator, addParameters, configure } from '@storybook/react';
+import { themes } from '@storybook/theming';
+import { ThemeDecorator } from '../src/stories/decorators';
+import { createGlobalStyle } from 'styled-components';
+import global from '@codesandbox/common/lib/global.css';
+
+const viewports = {
```
Comment (siddharthkp) [inline packages/components/.storybook/config.tsx]: can we import this from the design language as well?

File: packages/components/.eslintignore [hunk 5]
```diff
@@ -0,0 +1,3 @@
+src/forked-vendors
+src/stories
+*.stories.tsx
```
Comment (siddharthkp) [inline packages/components/.eslintignore]: we ignore stories? 🤔

File: package.json [hunk 6]
```diff
@@ -58,6 +61,7 @@
     "test:ci": "lerna run test --ignore codesandbox-browserfs -- --ci --testResultsProcessor=\"jest-junit\" ",
     "test:integrations": "lerna exec --scope app --stream -- yarn test:integrations",
     "test:jest-lite": "lerna exec --scope app --stream -- yarn run test jest-lite --watch --coverage",
+    "chromatic": "lerna run chromatic --scope=@codesandbox/components --stream",
```
Comment (siddharthkp) [inline package.json]: woot woot!

**Comment by siddharthkp** [comment]
This is great! Exciting times

File: packages/components/src/design-language/typography.ts (hunk 2 above)
Comment (SaraVieira) [reply packages/components/src/design-language/typography.ts]: Makes sense! Done :)

File: packages/components/package.json (hunk 3 above)
Comment (SaraVieira) [reply packages/components/package.json]: I mean it gets compiled, I don't see why

File: packages/components/.storybook/config.tsx (hunk 4 above)
Comment (SaraVieira) [reply packages/components/.storybook/config.tsx]: we don't really have this there :(

File: packages/app/src/app/pages/common/LikeHeart/index.tsx [hunk 7]
```diff
@@ -17,11 +17,11 @@ type Props = {
   colorless?: boolean;
   sandbox: Sandbox;
   text?: number;
-} & Partial<Pick<ComponentProps<typeof MaybeTooltip>, 'disableTooltip'>> &
-  Pick<
-    ComponentProps<typeof Container>,
-    'className' | 'highlightHover' | 'style'
-  >;
+  className?: string;
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/common/LikeHeart/index.tsx]: @SaraVieira Why did you change this back? 🤔
I used `Pick` so it's more clear what props we just pass through to the `Container` & `MaybeTooltip` components

File: packages/common/src/components/Preference/PreferenceNumber.tsx [hunk 8]
```diff
@@ -1,12 +1,13 @@
...
+import React, { ChangeEvent, FunctionComponent } from 'react';
 
 import { Input } from './elements';
 
 type Props = {
   setValue: (value: number) => void;
   step?: number;
   value: number;
-} & Pick<ComponentProps<typeof Input>, 'style'>;
+  style?: React.CSSProperties;
```
Comment (MichaelDeBoey) [inline packages/common/src/components/Preference/PreferenceNumber.tsx]: Same

File: packages/common/src/components/Preference/PreferenceText/index.tsx [hunk 9]
```diff
@@ -1,20 +1,17 @@
...
-} from 'react';
+import { ChangeEvent, createElement, FunctionComponent } from 'react';
 
 import { Input, TextArea } from './elements';
 
 type Props = {
   block?: boolean;
   isTextArea?: boolean;
   placeholder?: string;
+  style?: React.CSSProperties;
```
Comment (MichaelDeBoey) [inline packages/common/src/components/Preference/PreferenceText/index.tsx]: Same
